_gsocproposals/2020/proposal_IntAlertSystem.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Intelligent Alert Management System For HEP experiments
+layout: gsoc_proposal
+project: Monitoring
+year: 2020
+organization: CERN
+---
+
+## Description
+
+We would like to build an intelligent Alert Management System for HEP experiments.
```

Comment:
I'm missing a sentence of motivation in the beginning, explaining the general problem you are trying to solve and its context.